### ISSUE TITLE
grafana: no data  on Ubuntu after #8517

### ISSUE THREAD

**Issue by allnil** [paradigmxyz/reth#8603]
### Describe the bug

Grafana can't see the reth instance and shows NO DATA on dashboards using the latest reth image on Ubuntu 22.04 LTS
I identified that the issue first time occured in this PR commit  #8517
I build e89e4c92fd967ab39c5f0d9d76d31f088a79b944 and previous commit 80809a491887b770a29b1368b0c3712fec5d2464 locally to compare them and used docker-compose from etc/ folder
```
docker compose -f etc/docker-compose.yml -f etc/lighthouse.yml up -d
```

The node itself is working and the state is growing
<img width="772" alt="Screenshot 2024-06-04 at 15 15 55" src="https://github.com/paradigmxyz/reth/assets/27956082/22c7f687-5657-41c0-b7fd-c9aeb9210c21">
<img width="1442" alt="Screenshot 2024-06-04 at 15 49 08" src="https://github.com/paradigmxyz/reth/assets/27956082/84918f0a-16bf-41da-a547-7f0327a80d03">

### Steps to reproduce

1. docker compose -f etc/docker-compose.yml -f etc/lighthouse.yml up -d
with latest image on ubuntu
2. go to the grafana dashboard

### Platform(s)

Linux (x86)

### What version/commit are you on?

latest

### What database version are you on?

latest

### Which chain / network are you on?

mainnet

### What type of node are you running?

Archive (default)

### If you've built Reth from source, provide the full command you used

docker build . -t reth:local

### Code of Conduct

- [X] I agree to follow the Code of Conduct

**Comment by onbjerg** [comment]
hey! is this still the case on beta.9?

**Comment by mattsse** [comment]
pretty sure we fixed this via #8593

closing optimistically, please ping if this is still an issue

**Comment by allnil** [comment]
it works
checked!
